packages/artblocks/src/test-transactions.ts

```diff
@@ -0,0 +1,338 @@
+import { type MintActionParams } from '@rabbitholegg/questdk'
+
+import {
+  type TestCase,
+  type TestParams,
+  createTestCase,
+} from '@rabbitholegg/questdk-plugin-utils'
+
+import { NULL_ADDRESS } from './const'
```

Comment:
Just an fyi, viem has a built in constant called zeroAddress you can use for this purpose